README.md

```diff
@@ -1,16 +1,11 @@
 # 
-Portfolio-project Project.
```

Comment:
Please add all the files to the project from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.